packages/hash/api/src/model/knowledge/page.model.ts

```diff
@@ -0,0 +1,481 @@
+import { ApolloError, UserInputError } from "apollo-server-express";
+import { GraphApi } from "@hashintel/hash-graph-client";
+import { generateKeyBetween } from "fractional-indexing";
```

Comment:
We are making use of link indicies now instead of the property, right? I assume we can get rid of this

Reply:
I don't think so, I believe this is used for maintaining the position of the pages in the sidebar (so they are re-arrangeable).

@luisbettencourt, can you confirm the index stored on pages is needed?

Reply:
My bad, you're right :)